src/main/java/org/broadinstitute/hellbender/utils/io/IOUtils.java

```diff
@@ -564,14 +565,36 @@ public static void assertFileIsReadable(final File file) {
     public static void assertFileIsReadable(final Path path) {
         Utils.nonNull(path);
 
-        if ( ! Files.exists(path) ) {
-            throw new UserException.CouldNotReadInputFile(path, "It doesn't exist.");
-        }
-        if ( ! Files.isRegularFile(path) ) {
-            throw new UserException.CouldNotReadInputFile(path, "It isn't a regular file");
-        }
-        if ( ! Files.isReadable(path) ) {
-            throw new UserException.CouldNotReadInputFile(path, "It is not readable, check the file permissions");
+        try {
+            if ( ! Files.exists(path) ) {
+                throw new UserException.CouldNotReadInputFile(path, "It doesn't exist.");
+            }
+            if ( ! Files.isRegularFile(path) ) {
+                throw new UserException.CouldNotReadInputFile(path, "It isn't a regular file");
+            }
+            if ( ! Files.isReadable(path) ) {
+                throw new UserException.CouldNotReadInputFile(path, "It is not readable, check the file permissions");
+            }
+        } catch (com.google.cloud.storage.StorageException cloudBoom) {
+            // probably a permissions problem, or perhaps a disabled account.
+            logger.debug(cloudBoom.toString());
+            logger.debug("code: " + cloudBoom.getCode());
+            logger.debug("location: " + cloudBoom.getLocation());
+            logger.debug("reason: " + cloudBoom.getReason());
+            logger.debug("message: " + cloudBoom.getMessage());
+            String msg = "";
+            for (StackTraceElement x : cloudBoom.getStackTrace()) {
```

Comment:
Oh thank you, I didn't know about that option! I have removed the whole printing code then since it's unnecessary.